iothub/device/src/Transport/Mqtt/MqttTransportHandler.cs

```diff
@@ -357,8 +354,9 @@ public override async Task<Message> ReceiveAsync(TimeoutHelper timeoutHelper)
             }
         }
 
-        private Message ProcessMessage(Message message, bool hasMessage)
+        private Message ProcessMessage(bool hasMessage)
```

Comment:
This method was always called with null as the input for `message`. There is no reason to take it as an input since it can be super confusing. Refactored.